### ISSUE TITLE
Playsinline attribute is missing in video elements with autoplay enable

### ISSUE THREAD

**Issue by danramzdev** [soflyy/breakdance-bugs#696]
Breakdance: v1.2.1
Browser: Safari.
OS: iOS

**Reproduction  Steps**

Add a video element, load a video, in video options enable autoplay, once with that the video element is created but it does not contain the playsinline attribute to play correctly on iOS devices.

![Captura de pantalla 2023-02-16 202931](https://user-images.githubusercontent.com/32352812/219534418-b69373a3-cbbd-4240-8dc6-bb31687dfd15.png)

To make it play automatically on iOS I have to add the following code:
```
<script>
window.addEventListener("DOMContentLoaded", function () {
  const videos = document.querySelectorAll(".ee-video[autoplay]");
  for (let i = 0; i < videos.length; i++) {
    videos[i].setAttribute("playsinline", true);
    videos[i].play()
  }
});
</script>
```

**Comment by a-dubiel** [comment]
I will double check this. 

> Breakdance: v1.2.1
> Browser: Chrome.
> OS: Windows

This threw me off. Videos in Chrome work as expected.

**Comment by danramzdev** [comment]
Sorry, I made a mistake in putting that information, the problem is actually on iOS devices.

**Comment by louisreingold** [comment]
This is addressed in Breakdance 1.4.
